packages/fuels/tests/scripts.rs

```diff
@@ -298,13 +298,13 @@ async fn test_script_call_with_non_default_max_input() -> Result<()> {
 #[tokio::test]
 async fn test_script_raw_slice() -> Result<()> {
     abigen!(Script(
-        name = "BimBamScript",
```

Comment:
I would leave this one as `BimBamScript` and use the same name in the next test.